src/main/webapp/data/developers.json

```diff
@@ -221,6 +221,9 @@
       "name": "Bhargav Thanki",
       "username": "bmthanki"
     },
+    {
+      "username": "bhumisaxena0223"
```

Comment:
https://github.com/TEAMMATES/teammates/pull/9086#issuecomment-419650809

Reply:
Working on it 😄